packages/kovan-faucets/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@0xproject/erc20-token-faucet",
```

Comment:
This is an inaccurate name. It actually dispenses ZRX (ERC20) or Ether (Non-ERC20).